### PR TITLE
Fix entrypoint

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@RemiDesgrange : here are my changes to nextcloud-app

**Review by RemiDesgrange — APPROVED** [review]
LGTM :shipit:

File: extension/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -43,6 +43,7 @@ public function __construct(array $urlParams = []) {
     }
 
     public function register(IRegistrationContext $container): void {
```
Comment (RemiDesgrange) [inline extension/lib/AppInfo/Application.php]: So the method can be suppressed ?

File: extension/lib/AppInfo/Application.php [hunk 2]
```diff
@@ -10,10 +10,10 @@
 
 use OCA\Georchestra\User\Backend;
 use OCA\Georchestra\Service\UserService;
-use ReflectionObject;
+// use ReflectionObject;
```
Comment (RemiDesgrange) [inline extension/lib/AppInfo/Application.php]: Delete ?

File: extension/lib/User/Backend.php [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
 
 namespace OCA\Georchestra\User;
+
+/*
 use OC\User\Database;
 use OCP\User\Backend\ABackend;
 use OC\Log;
 use OCP\IUserBackend;
 use OCP\User\Backend\ICheckPasswordBackend;
+*/
```
Comment (RemiDesgrange) [inline extension/lib/User/Backend.php]: Delete ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+https://github.com/nextcloud/docker
+
+Debuggage VSCode:
+
+https://confluence.camptocamp.com/confluence/display/GEOR/NextCloud
+
+Postmortem:
+
+https://confluence.camptocamp.com/confluence/display/GEOR/Postmortem+Projet+Nextcloud
```
Comment (RemiDesgrange) [inline README.md:57]: you can't put internal link to public repos :smile:

Comment (arnaud-morvan) [reply README.md:57]: Maybe we can move it to wiki.

Comment (arnaud-morvan) [reply README.md:57]: Just would like to keep track of it, but yes I can remove them from the PR.

**Pull request merged**